reference/config_files/global_conf.rst

```diff
@@ -31,6 +31,34 @@ To list all the possible configurations available, run :command:`conan config li
 
 .. include:: ../../common/config_list.inc
 
+Description of configurations
++++++++++++++++++++++++++++++
+
+core.cache:storage_path
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Absolute path to a folder where the Conan packages and the database of the packages will be stored.
+This folder will take the most of the space in your disk, as it stores the binary pacakges dowloaded or created.
+
+.. code-block:: text
+    :caption: *global.conf*
+
+    core.cache.storage_path = C:\Users\danielm\my_conan_storage_folder
+
+**Default value:** ``<CONAN_HOME>/p``
+
+core.download:download_cache
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Absolute path to a folder where the Conan packages *uncompressed* will be stored.
```

Comment:
```suggestion
Absolute path to a folder where the Conan packages will be stored *uncompressed*.
```

But are you sure about the uncompressed? I though this were for the `conan_package.tar.gz` files and such

Reply:
sorry, I meant just the opposite 🤣 
```suggestion
Absolute path to a folder where the Conan packages will be stored *compressed*.
```

Reply:
:P